templates/helm-values.tpl.yaml

```diff
@@ -162,97 +162,27 @@ kube-prometheus:
             tlsConfig:
               caFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
               insecureSkipVerify: true
-    rules:
```

Comment:
This is now part of the default rules?

Reply:
yes, all the rules in the list are covered already